src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -328,13 +328,13 @@ static void usage(void)
     debugs(MY_DEBUG_SECTION, DBG_CRITICAL, "Wrong usage. Please reconfigure in squid.conf.");
 
     std::cerr <<
-              "Usage: " << program_name << " [-d] [-b dbpath] [-p pauselen] [-h] configfile\n"
-              "	-d            enable debugging output\n"
-              "	-l logfile    log messages to logfile\n"
-              "	-b dbpath     Path where persistent session database will be kept\n"
-              "	              If option is not used, then " DEFAULT_QUOTA_DB " will be used.\n"
-              "	-p pauselen   length in seconds to describe a pause between 2 requests.\n"
-              "	-h            show show command line help.\n"
+              "Usage: " << program_name << " [-d level] [-b dbpath] [-p pauselen] [-h] configfile\n" <<
+              "	-d level      set section " << MY_DEBUG_SECTION << " debugging to the specified level,\n"
+              "               overwriting Squid's debug_options (default: 1)\n"
+              "	-b dbpath     Path where persistent session database will be kept\n" <<
+              "	              If option is not used, then " DEFAULT_QUOTA_DB " will be used.\n" <<
```

Comment:
It is not out of scope of this PR because this PR already changes all these lines for the same reason -- introduction of "<<" operators. If unnecessary introduction of "<<" operators is out of scope, then this PR should be blocked for containing out of scope changes!

Reply:
@kinkie, I do insist on this trivial change (or a different reason for avoiding it). AFAICT, doing a followup PR to add two "<<" operators to lines that already gained those operators in this PR is wasteful.

Reply:
Oh. I had misunderstood that you were suggesting changing the type of DEFAULT_QUOTA_DB in this PR.
Given that, of course , no problem adding the operators. coming up shortly